modules/aaxBlockmeterRtdProvider.md

```diff
@@ -0,0 +1,48 @@
+## Overview
+
+Module Name: AAX Blockmeter Realtime Data Module  
+Module Type: Rtd Provider  
+Maintainer: product@aax.media  
+
+## Description
+
+The module enables publishers to measure traffic coming from visitors using adblockers. 
```

Comment:
I think this requires an account with AAX yes? Might be better to state that here, as it doesn't look like the module itself does much more than load the AAX script.